src/model/Clearing.java

```diff
@@ -0,0 +1,31 @@
+package model;
+
+public class Clearing {
+    String type;
+    String dwelling; // not sure what type this should actually be
+    String location; // indicating the location on the board - should this be int?
```

Comment:
@sstone93 @michaelcatt said a Pair would probably be best. This way the top left corner would be (0,0), and then you increment up or down to move from there.
But we wanted to know what you wanted done here.